### PR TITLE
C# v4 compatible examples

### PR DESCRIPTION
**Pull request by nwithan8**
- New examples written in v4 syntax
- Fix bad example for brand update in v3

### CODE REVIEW

File: official/docs/csharp/current/carrier_accounts/update-carrier-account.cs [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        {
+            string apiKey = Environment.GetEnvironmentVariable("EASYPOST_API_KEY")!;
+
+            var client = EasyPost.Client(apiKey);
+
+            CarrierAccount carrierAccount = await client.CarrierAccount.Retrieve("ca_...");
+
+            await carrierAccount.Update(new Dictionary<string, object>()
+            {
+                { "description", "FL Location UPS Account" }
```
Comment (jchen293) [inline official/docs/csharp/current/carrier_accounts/update-carrier-account.cs:22]:
```suggestion
                { "description", "FL Location UPS Account" },
```

File: official/docs/csharp/current/customs_info/create-customs-info.cs [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+            {
+                { "customs_certify", true },
+                { "customs_signer", "Steve Brule" },
+                { "contents_type", "merchandise" },
+                { "contents_explanation", "" },
+                { "restriction_type", "none" }
+                {
+                "eel_pfc",
+                "NOEEI 30.37(a)"
+            },
```
Comment (jchen293) [inline official/docs/csharp/current/customs_info/create-customs-info.cs:28]: Missing comma and need to format line 25-28

File: official/docs/csharp/current/customs_items/create-customs-item.cs [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+            var client = EasyPost.Client(apiKey);
+
+            CustomsItem customsItem = await client.CustomsItem.Create(new Dictionary<string, object>()
+            {
+                { "description", "T-shirt" },
+                { "quantity", 1 },
+                { "weight", 5 },
+                { "value", 10 },
+                { "hs_tariff_number", "123456" },
+                { "origin_country", "US" },
```
Comment (jchen293) [inline official/docs/csharp/current/customs_items/create-customs-item.cs:25]:
```suggestion
                { "origin_country", "US" }
```

File: official/docs/csharp/current/orders/create-order.cs [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+                                }
+                            },
+                            {
+                                new Dictionary<string, object>
+                                {
+                                    {
+                                        "parcel", new Dictionary<string, object>()
+                                        {
+                                            { "predefined_package", "FedExBox" },
+                                            { "weight", 17.5 },
```
Comment (jchen293) [inline official/docs/csharp/current/orders/create-order.cs:54]:
```suggestion
                                            { "weight", 17.5 }
```

File: official/docs/csharp/current/pickups/create-pickup.cs [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+            Pickup pickup = await client.Pickup.Create(new Dictionary<string, object>()
+            {
+                { "reference", "my-first-pickup" },
+                { "min_datetime", "2022-10-01 10:30:00" },
+                { "max_datetime", "2022-10-01 17:30:00" },
+                { "shipment", "shp_..." },
+                { "address", "adr_..." },
+                { "is_account_address", false },
+                { "instructions", "Special pickup instructions" },
```
Comment (jchen293) [inline official/docs/csharp/current/pickups/create-pickup.cs:26]:
```suggestion
                { "instructions", "Special pickup instructions" }
```

File: official/docs/csharp/current/carrier_accounts/update-carrier-account.cs (hunk 1 above)
Comment (nwithan8) [reply official/docs/csharp/current/carrier_accounts/update-carrier-account.cs:22]: Resolved

File: official/docs/csharp/current/customs_info/create-customs-info.cs (hunk 2 above)
Comment (nwithan8) [reply official/docs/csharp/current/customs_info/create-customs-info.cs:28]: Resolved

File: official/docs/csharp/current/customs_items/create-customs-item.cs (hunk 3 above)
Comment (nwithan8) [reply official/docs/csharp/current/customs_items/create-customs-item.cs:25]: Resolved

File: official/docs/csharp/current/orders/create-order.cs (hunk 4 above)
Comment (nwithan8) [reply official/docs/csharp/current/orders/create-order.cs:54]: Resolved

File: official/docs/csharp/current/pickups/create-pickup.cs (hunk 5 above)
Comment (nwithan8) [reply official/docs/csharp/current/pickups/create-pickup.cs:26]: Resolved

**Review by jchen293 — APPROVED** [review]

**Comment by nwithan8** [comment]
> I trust that these are all identical with the addition of the `client` object?

yes, pretty much a find and replace for the client initialization, and then calling the method off the client object.

**Pull request merged**